plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverApplication.java

```diff
@@ -547,12 +548,15 @@ protected void initializeConfiguration() {
     }
 
     /**
-     * May be overrided in implementors
+     * May be override in implementors
      */
     protected void initializeApplication() {
 
     }
 
+    protected void initProductLicenseList() {
```

Comment:
No need to create separate function, especially in CE. Move it  in initializeApplicationServices